element_array_ephys/export/nwb/nwb.py

```diff
@@ -0,0 +1,600 @@
+import os
+import decimal
+import json
+import numpy as np
+import pynwb
+import datajoint as dj
+from element_interface.utils import find_full_path
+from hdmf.backends.hdf5 import H5DataIO
+from hdmf.data_utils import GenericDataChunkIterator
+from nwb_conversion_tools.tools.nwb_helpers import get_module
+from nwb_conversion_tools.tools.spikeinterface.spikeinterfacerecordingdatachunkiterator import (
+    SpikeInterfaceRecordingDataChunkIterator
+)
+from spikeinterface import extractors
+from tqdm import tqdm
+
+from ... import probe, ephys_acute, ephys_chronic, ephys_no_curation
```

Comment:
```suggestion
from ... import probe, ephys_acute
```

Reply:
Removing `ephys_no_curation` since that pull request is not yet merged.  I am not entirely sure if the `ephys_chronic` module will work since the foreign key references are slightly different.  Created issue #55 to test these export functions for `ephys_chronic` in the future.